launcher/ui/pages/modplatform/ImportPage.cpp

```diff
@@ -149,7 +149,7 @@ void ImportPage::setUrl(const QString& url)
 void ImportPage::on_modpackBtn_clicked()
 {
     auto filter = QMimeDatabase().mimeTypeForName("application/zip").filterString();
-    filter += ";;" + tr("Modrinth pack (*.mrpack)");
+    filter += ";;" + tr("Modrinth pack (*.mrpack *mrpack)");
```

Comment:
Okay I looked at how other applications do this and it seems like they just put the filter outside of the translation:
```suggestion
    filter += ";;" + tr("Modrinth pack") + " (*.mrpack)";
```

I just tested this, and it works (I manually translated the new string and added a local translation to my development installation)

I am going to push a commit here. Feel free to object!

Reply:
makes sense :D